src/plugins/application_config/server/routes/index.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  IRouter,
+  IScopedClusterClient,
+  Logger,
+  OpenSearchDashboardsResponseFactory,
+} from '../../../../core/server';
+import { ConfigurationClient } from '../types';
+
+export function defineRoutes(
+  router: IRouter,
+  getConfigurationClient: (inputOpenSearchClient: IScopedClusterClient) => ConfigurationClient,
+  logger: Logger
+) {
+  router.get(
+    {
+      path: '/api/config/exists',
```

Comment:
why do we need specific` exists` api

Reply:
Answered here https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5641#discussion_r1439933024

Reply:
Removed the `exists` API. It was originally motivated by OpenSearch implementation detail.